### DISCUSSION TITLE
New feature on add user

### DISCUSSION THREAD

**Discussion by JoanFo1456** [pelican-dev/panel#570] [General]
It would be great if you want to add a user with all perms on a server, add a new button so you can mark everything instead of having to click all the all perms on the categories.
It would be on the lower part and left, so you can't click accidentally, or add a confirm, would be less annoying i think.

**Comment by Boy132** [comment]
Will be added in the new client area:
![grafik](https://github.com/user-attachments/assets/1ecf2394-73c9-45b0-98c9-4239c2217bd5)
